modules/creating-a-github-application-in-administrator-perspective.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="creating-a-github-application-in-administrator-perspective_{context}"]
+
+= Creating a GitHub application in administrator perspective
+
+As a cluster administrator, you can configure your GitHub application with the OpenShift Container Platform cluster to use pipeline as code. This allows you to execute a set of tasks required for build deployment.
+
+.Prerequisites
+You have installed the Red Hat OpenShift pipelines 1.8 operator from the custom catalog source
+
+.Procedure
+. In the administrator perspective, navigate to *Pipelines* using the left navigation pane.
```

Comment:
for usability, avoid directional words like `left, right, up, down` 

simply say, `...using the navigation pane.`